### PR TITLE
22549 Small message box UI tweaks

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#22549

*Description of changes:*
- app version = 5.11.11
- adjust message box icon size, font size, and whitespace

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

**Comment by severinbeauvais** [comment]
Single-line example:

image

Multi-line example:

image

File: src/components/ContinuationIn/ContinuationAuthorization.vue [hunk 1]
```diff
@@ -403,7 +406,7 @@ textarea {
   // ref: https://developer.mozilla.org/en-US/docs/Web/CSS/field-sizing
   // field-sizing: content;
   // unset height as it's sometimes 0 initially
-  height: unset !important;
```
Comment (severinbeauvais) [inline src/components/ContinuationIn/ContinuationAuthorization.vue:406]: `!important` overrode the component's own height specification (set by the v-auto-resize directive). It works perfectly with the new code.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-create-dev--pr-724-m4j6eaqw.web.app

File: src/components/ContinuationIn/ContinuationAuthorization.vue [hunk 2]
```diff
@@ -152,7 +155,7 @@
             <textarea
               id="textarea-message"
               v-auto-resize
-              class="font-16 font-italic ml-8"
+              class="font-14 ml-6 pr-5 mb-n2"
```
Comment (severinbeauvais) [inline src/components/ContinuationIn/ContinuationAuthorization.vue:158]: The textarea adds some whitespace below the text itself, which you can see here:

image

`mb-n2` pulls up the bottom of the textarea ("shrinks it") to remove the whitespace:

image

**Review by ArwenQin — APPROVED** [review]
Looks good!

**Pull request merged**